lib/ruby2d/window.rb

```diff
@@ -26,14 +27,19 @@ def initialize(args = {})
       @vsync       = args[:vsync]  || true
       @mouse_x, @mouse_y = 0, 0
       @objects     = []
-      @key, @key_down, @key_held, @key_up = [], [], [], []
-      @mouse, @mouse_down, @mouse_up, @mouse_scroll, @mouse_move = [], [], [], [], []
-      @controller, @controller_axis = [], []
-      @controller_button_down, @controller_button_up = [], []
+      @key, @key_down, @key_held, @key_up = {}, {}, {}, {}
+      @mouse, @mouse_down, @mouse_up, @mouse_scroll, @mouse_move = {}, {}, {}, {}, {}
+      @controller, @controller_axis = {}, {}
+      @controller_button_down, @controller_button_up = {}, {}
+      @event_key = "a"
```

Comment:
I was thinking about, I even added comment about this; ). Problem I'm worried about is that if somebody removes a lot of event handlers, it might eventually move into the area of bigints, which are not present in MRuby. When this happens all sort of weird behavior might happen. We don't have this problem with strings.